### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by AbbyNyakara**
complete the main page

### CODE REVIEW

**Review by danielkryska — COMMENTED** [review]
Hi Abby,
I'd like to join your project as a code reviewer. As the project is very simple there isn't too much to be checked in a matter of code itself. Although there are also other things that need to be done properly.

**Commits titles and description** 
- Titles of commits should tell "future you" as also other developers what the commit covers in short words. It should be done in an active voice (starting with words like add, create, delete, update, ...). The commit titles should be clear on what you're working on (for example `add files` isn't obvious).
  Some of your commits cover the convention, while others do not.
image
- if there are many little changes in your commit you should also include a list of them in the description, like

`complete main page` should be changed into

```
Improvement of the main page
- fix footer overlapping
- add a new header section
- fix ...
- remove ...

```

**Comment by danielkryska** [comment]
**Branching**
I'm glad to see you're using `git workflow` with `main` and `dev` branches. :) In the case of your `feature branches,` you should try to use names that will cover the main idea of your work. For example, your current `pull request` doesn't cover only `setup`. The branch covers also basic layout creation. Therefore `init-app-and-create-basic-layout` would be recommended by me. Or even split both of the issues into two separate branches called `initial-setup` and `create-basic-layout`.

**Comment by danielkryska** [comment]
As a good place to work with commits messages clarification I'd recommend:

- https://reflectoring.io/meaningful-commit-messages/
- https://blog.devgenius.io/a-guide-to-writing-good-commit-messages-in-git-4e513e887b6b

There is also a very good technic for separating concerns while writing commits. You can get familiar with it:
- documentation of [Conventional commiting](https://www.conventionalcommits.org/en/v1.0.0/)
- https://medium.com/swlh/writing-better-commit-messages-9b0b6ff60c67

**Comment by danielkryska** [comment]
Here catch a good convention for branch naming:
- https://dpericich.medium.com/how-to-write-great-git-branch-names-to-improve-your-team-3345aee7fc2a

File: src/components/Footer/footer.scss [hunk 1]
```diff
@@ -1,10 +1,29 @@
...
+
+.footer__left {
+  color: variables.$clr-grey;
+  display: flex;
+  gap: 7rem;
+}
+
+.footer__socials {
+  display: flex;
+  gap: 1.5rem;
```
Comment (danielkryska) [inline src/components/Footer/footer.scss:27]: Good for you, that you're using BEM. :)

Comment (AbbyNyakara) [reply src/components/Footer/footer.scss:27]: Awesome. Ill take in all the changes in my next feature.

**Pull request merged**